### ISSUE TITLE
Unclear on multiple Vugu files

### ISSUE THREAD

**Issue by charles-d-burton** [vugu/vugu#162]
**Question**
I don't fully grok on how to create/use multiple .vugu files.  I'm certain that it's going to require some routing, but I can't quite seem to get them to be created.  I have a root.vugu that works as I would expect but when I try to add a new signup.vugu with empty contents I'm getting an error.

```go
<div class="some-name">
<!-- ... -->
</div>

<script type="application/x-go">
// Go code here
</script>
```

**Is your question related to a problem? Please describe.**
I think it's just documentation really.  I'm not really sure how to get new .vugu files added to stub out a system.

**Suggested Changes**
Add a complete example with two .vugu files.

**Comment by bradleypeabody** [comment]
Yeah this is mostly a documentation problem.  I'm working on some things that I hope will improve the situation a lot.  

Meanwhile what you're looking for should be, for the most part, here: https://www.vugu.org/doc/routing

The idea is you set up a router to correlate which page (vugu component) is assigned to a variable and display in the middle of your root components via a vg-comp tag.

**Comment by charles-d-burton** [comment]
So, I'm assuming I do something like:
```html
<div>
  <vg-comp expr="c.Signup"></vg-comp>
</div>
```

**Comment by bradleypeabody** [comment]
More like:

root.vugu:
```
<div>
  <!-- other general page header stuff can go here -->
  <vg-comp expr="c.Body"></vg-comp>
  <!-- footer stuff, etc. -->
</div>

<script type="application/x-go">
type Root struct {
    Body vugu.Builder
}
</script>
```

signup.vugu
```
```

**Comment by owenwaller** [comment]
If this hasn't been documented on the vugu.org website can we update the documentation before v1.0?
